### PR TITLE
Merge GEOS-Chem 14.4.3 into the 14.5.0 development branch

### PR DESCRIPTION
**Pull request by yantosca**
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Describe the update
This PR seeks to merge the updates from the 14.4.3 release into the 14.5.0 development branch.  

### Expected changes
This is a no-diff-to-benchmark PR (i.e. the concentrations in the fullchem restart files are identical w/r/t 14.4.2)  But some diagnostic outputs may differ due to the updates contained in these PRs:
- #2394 
- #2399
- #2410
- #2402

### Reference(s)
N/A

### Related Github Issue
N/A

Tagging @msulprizio @lizziel

### CODE REVIEW

**Comment by yantosca** [comment]
Integration tests are running

**Comment by yantosca** [comment]
The GEOS-Chem "alldiags" integration test failed with this error:
```console
Error termination. Backtrace:
At line 1183 of file /n/holyscratch01/jacob_lab/ryantosca/tests/14.5.0/gc2425/GCC_it/CodeDir/src/GEOS-Chem/GeosCore/diagnostics_mod.F90
Fortran runtime error: Index '264' of dimension 3 of array 'colmass' above upper bound of 237
```
All other tests passed.  I am investigating.

**Comment by yantosca** [comment]
The error in the "alldiags" test has been traced to the ALK4P species being listed in `species_database.yml` as an advected species, but omitted from the list of advected species in `geoschem_config.yml`.  A fix will follow in PR #2318.

**Comment by yantosca** [comment]
As mentioned above, all GEOS-Chem Classic integration tests (except "alldiags") passed.  The issue with alldiags will be fixed in PR #2318, which will be merged into GEOS-Chem 14.5.0 after this PR.
```console
==============================================================================
GEOS-Chem Classic: Execution Test Results

CodeDir   : c2a3236 Merge GEOS-Chem 14.4.3 back into the 14.5.0 development branch
GEOS-Chem : 000205c86 Merge GEOS-Chem 14.4.3 back into the 14.5.0 development branch
HEMCO     : 41605bd HEMCO 3.9.3 release
Cloud-J   : d20050f Update version to 7.7.3
HETP      : 2a99b24 Merge pull request #2 from geoschem/bugfix/initialize_local_variables

Using 24 OpenMP threads
Number of execution tests: 30

Submitted as SLURM job: 43708487
==============================================================================
...
gc_4x5_merra2_fullchem_alldiags.....................Execute Simulation....FAIL
...
Summary of test results:
------------------------------------------------------------------------------
Execution tests passed: 29
Execution tests failed: 1
Execution tests not yet completed: 0
```

**Comment by yantosca** [comment]
All GCHP integration tests passed:
```console
==============================================================================
GCHP: Execution Test Results

CodeDir       : 46197b6 Merge GEOS-Chem 14.4.3 back into the 14.5.0 development branch
MAPL          : 231d53c Merge pull request #36 from geoschem/feature/improve_hflux_regridding
GMAO_Shared   : 4ddb3ec Merge pull request #2 from geoschem/feature/mapl-upgrade
ESMA_cmake    : ad5deba Added ecbuild as a submodule of ESMA_cmake
gFTL-shared   : 4b82492 Merge branch 'upstream_v1.5.0' into feature/v1.5.0
FMS           : 259759d Merge pull request #3 from geoschem/feature/update_gmao_libs
FVdycoreCubed : af42462 Merge PR #8 (Add PLEadv diagnostic for offline advection in GCHP)
geos-chem     : 000205c86 Merge GEOS-Chem 14.4.3 back into the 14.5.0 development branch
HEMCO         : 41605bd HEMCO 3.9.3 release
yaFyaml       : 19afe50 Merge branch 'upstream_v1.0.4' into feature/v1.0.4
pFlogger      : 2c4b724 Merge branch 'upstream_v1.9.1' into feature/v1.9.1
Cloud-J       : d20050f Update version to 7.7.3
HETP          : 2a99b24 Merge pull request #2 from geoschem/bugfix/initialize_local_variables

Number of execution tests: 12

Submitted as SLURM job: 43793675
==============================================================================
 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%  All execution tests passed!  %%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
The reason why the "alldiags" test did not fail is because in GCHP we use only the SO4 species (which is advected, emitted, dry deposits, and wet deposits). 
```console
#==============================================================================
# Budget defined as species kg/s in the column (full, troposphere, or PBL)
# due to a single component (e.g. chemistry); default = SO4 only; add more
# species as needed (advected only)
#==============================================================================
  Budget.template:        '%y4%m2%d2_%h2%n2z.nc4',
  Budget.format:          'CFIO',
  Budget.timestampStart:  .true.
  Budget.monthly:         0
  Budget.frequency:       010000
  Budget.duration:        010000
  Budget.mode:            'time-averaged'
  Budget.fields:          'BudgetWetDepFull_SO4         ', 'GCHPchem',
                          'BudgetChemistryFull_SO4      ', 'GCHPchem',
                          'BudgetConvectionFull_SO4     ', 'GCHPchem',
                          'BudgetMixingFull_SO4         ', 'GCHPchem',
                          'BudgetEmisDryDepFull_SO4     ', 'GCHPchem',
                          #
                          'BudgetWetDepLevs1to35_SO4    ', 'GCHPchem',
                          'BudgetChemistryLevs1to35_SO4 ', 'GCHPchem',
                          'BudgetConvectionLevs1to35_SO4', 'GCHPchem',
                          'BudgetMixingLevs1to35_SO4    ', 'GCHPchem',
                          'BudgetEmisDryDepLevs1to35_SO4', 'GCHPchem',
                           #
                          'BudgetWetDepTrop_SO4         ', 'GCHPchem',
                          'BudgetChemistryTrop_SO4      ', 'GCHPchem',
                          'BudgetConvectionTrop_SO4     ', 'GCHPchem',
                          'BudgetMixingTrop_SO4         ', 'GCHPchem',
                          'BudgetEmisDryDepTrop_SO4     ', 'GCHPchem',
                           #
                          'BudgetWetDepPBL_SO4          ', 'GCHPchem',
                          'BudgetChemistryPBL_SO4       ', 'GCHPchem',
                          'BudgetConvectionPBL_SO4      ', 'GCHPchem',
                          'BudgetMixingPBL_SO4          ', 'GCHPchem',
                          'BudgetEmisDryDepPBL_SO4      ', 'GCHPchem',
::
```
The error in GCClassic is caused by the ALK4 species not being listed as an advected species, but being included in the Budget diagnostic.

File: CHANGELOG.md [hunk 1]
```diff
@@ -428,6 +449,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Removed State_Met%LWI and input meteorology LWI from carbon simulation run config files
 - Removed function `CLEANUP_UCX`; deallocations are now done in `state_chm_mod.F90`
 
+## [14.4.3] - 2024-08-13
```
Comment (msulprizio) [inline CHANGELOG.md:452]: Why are these lines so far down? Perhaps a merge issue? I think they need to be moved up.

**Review by msulprizio — CHANGES_REQUESTED** [review]
These updates look good for merging after fixing potential ordering issues in CHANGELOG.md.

File: CHANGELOG.md (hunk 1 above)
Comment (yantosca) [reply CHANGELOG.md:452]: Thanks Melissa, I can fix that in a post-merge fix.

Comment (msulprizio) [reply CHANGELOG.md:452]: OK. I will approve then.

**Review by msulprizio — APPROVED** [review]

**Pull request merged**